ZenPacks/zenoss/OpenStackInfrastructure/zenpack.yaml

```diff
@@ -33,7 +33,8 @@ class_relationships:
 # Cinder
 - Tenant 1:M Volume
 - Tenant 1:M VolSnapshot
-- Tenant 1:M Backup
+# release 2.2.0 does not do backup
+#- Tenant 1:M Backup
```

Comment:
Did we ever make a release that did have this "Tenant 1:M Backup" relationship? If so, we can't remove it or existing installs will be broken.